include/templatepublicclass.h

```diff
@@ -0,0 +1,31 @@
+/*
+ *   Copyright (C) 2017 Pelagicore AB
+ *
+ *   This Source Code Form is subject to the terms of the Mozilla Public
+ *   License, v. 2.0. If a copy of the MPL was not distributed with this
+ *   file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ *   SPDX-License-Identifier: MPL-2.0
+ *
+ */
+
+#ifndef TEMPLATEPUBLICCLASS_H_
+#define TEMPLATEPUBLICCLASS_H_
+
+#include <memory>
+
+class TemplatePrivateClass;
+
+class TemplatePublicClass
+{
+public:
+    TemplatePublicClass();
```

Comment:
Perhaps you could add a `helloWorld()` method which would return a "hello world" string or something which we then could call from the service which uses this library?